org.freedesktop.Sdk.Extension.node10.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: org.freedesktop.Sdk.Extension.node10
+branch: '18.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /usr/lib/sdk/node10
+  prepend-path: /usr/lib/sdk/node10/bin
+  strip: true
+modules:
+  - name: python2
+    config-opts:
+      - '--enable-static'
+      - '--disable-shared'
+      - '--with-system-expat'
+      - '--with-system-ffi'
+      - '--with-dbmliborder=gdbm'
+      - '--enable-unicode=ucs4'
+    sources:
+      - type: archive
+        url: https://www.python.org/ftp/python/2.7.16/Python-2.7.16.tgz
+        sha256: 01da813a3600876f03f46db11cc5c408175e99f03af2ba942ef324389a83bad5
+    cleanup:
+      - '/bin/2to3*'
+      - '/bin/easy_install*'
+      - '/bin/idle*'
+      - '/bin/pyvenv*'
+      - '/bin/pydoc*'
+      - '/include'
+      - '/share'
+
+        # Test scripts
+      - '/lib/python*/test'
+      - '/lib/python*/*/test'
+      - '/lib/python*/*/tests'
+      - '/lib/python*/lib-tk/test'
+      - '/lib/python*/lib-dynload/_*_test.*.so'
+      - '/lib/python*/lib-dynload/_test*.*.so'
+
+        # Unused modules
+      - '/lib/python*/idlelib'
+      - '/lib/python*/tkinter*'
+      - '/lib/python*/turtle*'
+      - '/lib/python*/lib2to3*'
+
+      # Static library
+      - '/lib/python2.7/config/libpython2.7.a'
+
+  - name: node
+    sources:
+      - type: archive
+        url: https://nodejs.org/dist/v10.15.3/node-v10.15.3.tar.gz
+        sha256: db460a63d057ac015b75bb6a879fcbe2fefaaf22afa4b6f6445b9db61ce2270d
+
+  - name: scripts
+    buildsystem: simple
+    build-commands:
+      - 'cp enable.sh install-node.sh install-full.sh /usr/lib/sdk/node10'
+      - 'mkdir -p /usr/lib/sdk/node10/share/metainfo'
+      - 'cp org.freedesktop.Sdk.Extension.node10.appdata.xml /usr/lib/sdk/node10/share/metainfo'
+      - 'appstream-compose --basename=org.freedesktop.Sdk.Extension.node10 --prefix=${FLATPAK_DEST} --origin=flatpak org.freedesktop.Sdk.Extension.node10'
+    sources:
+      - type: script
+        dest-filename: install-full.sh
+        commands:
+          - 'cp -r /usr/lib/sdk/node10 /app/node'
+      - type: script
+        dest-filename: install-node.sh
+        commands:
+          - 'mkdir -p /app/node/{bin,include,lib}'
```

Comment:
Why the install prefix is `/app/node` and not just `/app`? Also, `$FLATPAK_DEST` could be better - it's compatible with extensions and can be overriden.

Reply:
/app/node was used mostly just for consistency with the other SDK extensions, they always install to some subdirectory of /app.